pkg/scheduler/plugins/predicates/predicates_test.go

```diff
@@ -84,97 +57,41 @@ func TestEventHandler(t *testing.T) {
 	// nodes
 	n1 := util.BuildNode("node1", api.BuildResourceList("4", "4k", []api.ScalarResource{{Name: "pods", Value: "10"}}...), map[string]string{"selector": "worker"})
 	n2 := util.BuildNode("node2", api.BuildResourceList("3", "3k", []api.ScalarResource{{Name: "pods", Value: "10"}}...), map[string]string{})
-	n1.Status.Allocatable["pods"] = resource.MustParse("15")
-	n2.Status.Allocatable["pods"] = resource.MustParse("15")
 	n1.Labels["kubernetes.io/hostname"] = "node1"
 	n2.Labels["kubernetes.io/hostname"] = "node2"
 
 	// priority
 	p1 := &schedulingv1.PriorityClass{ObjectMeta: metav1.ObjectMeta{Name: "p1"}, Value: 1}
 	p2 := &schedulingv1.PriorityClass{ObjectMeta: metav1.ObjectMeta{Name: "p2"}, Value: 2}
 	// podgroup
-	pg1 := &schedulingv1beta1.PodGroup{
-		ObjectMeta: metav1.ObjectMeta{
-			Namespace: "ns1",
-			Name:      "pg1",
-		},
-		Spec: schedulingv1beta1.PodGroupSpec{
-			Queue:             "q1",
-			MinMember:         int32(2),
-			PriorityClassName: p2.Name,
-		},
-	}
-	pg2 := &schedulingv1beta1.PodGroup{
-		ObjectMeta: metav1.ObjectMeta{
-			Namespace: "ns1",
-			Name:      "pg2",
-		},
-		Spec: schedulingv1beta1.PodGroupSpec{
-			Queue:             "q1",
-			MinMember:         int32(1),
-			PriorityClassName: p1.Name,
-		},
-	}
+	pg1 := util.BuildPodGroupWithPrio("pg1", "ns1", "q1", 2, nil, schedulingv1beta1.PodGroupInqueue, p2.Name)
+	pg2 := util.BuildPodGroupWithPrio("pg2", "ns1", "q1", 1, nil, schedulingv1beta1.PodGroupInqueue, p1.Name)
+
 	// queue
-	queue1 := &schedulingv1beta1.Queue{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: "q1",
-		},
-	}
+	queue1 := util.BuildQueue("q1", 0, nil)
 
 	// tests
-	tests := []struct {
-		name     string
-		pods     []*apiv1.Pod
-		nodes    []*apiv1.Node
-		pcs      []*schedulingv1.PriorityClass
-		pgs      []*schedulingv1beta1.PodGroup
-		expected map[string]string
-	}{
+	tests := []uthelper.TestCommonStruct{
```

Comment:
every test case will initialize a new scheduler cache, but the origin test has a global schdeuler cache, is there any effect?

Reply:
No effect. There is just one case now.